client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java

```diff
@@ -32,9 +32,9 @@
 
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String messageBoxLocator = "android:id/message";
-    private String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
+    private final String messageBoxLocator = "android:id/message";
+    private final String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1275120317/issues/source?bid=2928425&fileBranchId=2943483#l36)